JetAnalyzers/python/Defaults_cff.py

```diff
@@ -76,7 +76,7 @@
     dobalance         = cms.bool(False),
     doflavor          = cms.bool(False),
     noabsflavors      = cms.bool(False),
-    drmax             = cms.double(0.3),
+    drmax             = cms.double(0.4),
```

Comment:
I would imagine this should match the value in the `JetResponseParameters`, so keep that in mind when answering my previous comment.